README.md

```diff
@@ -68,23 +68,38 @@ zuul:
     enabled: true 
     repository: REDIS 
     behind-proxy: true
-    default-policy: #optional - will apply unless specific policy exists
```

Comment:
default policy shouldn't be removed, this is required functionality

Reply:
We can choose to support the default policy with the policy default configuration option, and then select the policy that will be true by default when filtering policies.How's that?